sn_node/src/node/core/messaging/handling/handover.rs

```
@@ -54,6 +95,12 @@ impl Node {
             signed_vote
         );
 
+        if let Err(err) = self.check_signed_vote_saps(&signed_vote) {
+            // NB TODO tracking who sent this invalid vote?
+            error!("Ignoring invalid handover vote: {}", err);
```

Let's return an error here. I think we can then look to hook into dysfunction in a layer above.